### DISCUSSION TITLE
Looking to replicate this effect

### DISCUSSION THREAD

**Discussion by grdiweb** [matteobruni/tsparticles#4639] [Ideas]
Hello Matteo and the rest of collaborators,

I've switch to tsparticles coming from particle.js, now I have a very specific task which I believe is doable with tsparticles but I don't know how to code, I've tried by looking at different presets and samples but I am still not getting the result I want. I know this is not a request nor there is a place to do them, but this shouldn't be too hard and if anyone thinks it would look cool and be useful for other people in the future, feel free to try to make what I want, I for sure would use and be thankful.

I need this kind of effect for a website:
https://www.youtube.com/watch?v=Qghjf7y64D0

As you can see, they are embers falling down. To "try" to do this, I started using the Snow preset, changed directions, colors and swapped the circles with a strip image but it didn't look too good, I need the embers to rotate as in the "Multiple Images" preset but couldn't get it to work, also adding any kind of shadow / glowing effect... whatever is needed to make it look like the video above but in tsParticles.

I wish and hope anyone try to accomplish this and share the result so I and anyone could possibly use it. Thanks!

**Comment by matteobruni** [comment]
I think this is closest enough: https://codepen.io/matteobruni/pen/bGvzWXg

The shadow/glow has really bad performance in canvas 2d so I recommend using an image instead.

**Reply by grdiweb** [reply]
That's perfect!!! Thank you so so much <3

**Reply by grdiweb** [reply]
EDIT: I managed to place your code on my site by copying from "background:" till the end and replacing other tsParticles preset that was working fine. Thank you again! 

___________Ignore now:
For some reason I can't get it to work, sorry for the inconvenience. I have used other particle.js or tsParticles.js presets without issue, 

I place the JS script inside the page and put the ID tsparticles on where I want it displayed, also have a HTML code to load from CDN:

<script src="https://cdn.jsdelivr.net/particles.js/2.0.0/particles.min.js"></script>

but that won't do with this sample you made. I am using WordPress (Divi builder). I tried with different workarounds also just replacing the "particles" (and beyond) settings, but it won't show. Sorry for being such a noob to implement this. You spent more than enough time on this so I'm thankful anyway.

**Reply by matteobruni** [reply]
> EDIT: I managed to place your code on my site by copying from "background:" till the end and replacing other tsParticles preset that was working fine. Thank you again!
> 
> ___________Ignore now: For some reason I can't get it to work, sorry for the inconvenience. I have used other particle.js or tsParticles.js presets without issue,
> 
> I place the JS script inside the page and put the ID tsparticles on where I want it displayed, also have a HTML code to load from CDN:
> 
> <script src="https://cdn.jsdelivr.net/particles.js/2.0.0/particles.min.js"></script>
> but that won't do with this sample you made. I am using WordPress (Divi builder). I tried with different workarounds also just replacing the "particles" (and beyond) settings, but it won't show. Sorry for being such a noob to implement this. You spent more than enough time on this so I'm thankful anyway.

You have to use tsParticles script, not particles.js. It won't work with that.
